front/next.config.js

```diff
@@ -5,7 +5,7 @@ const CONTENT_SECURITY_POLICIES = [
   `script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googletagmanager.com *.google-analytics.com;`,
   `style-src 'self' 'unsafe-inline' *.typekit.net;`,
   `img-src 'self' data: https:;`,
-  `connect-src 'self' *.google-analytics.com;`,
+  `connect-src 'self' *.google-analytics.com *.jsdelivr.net;`,
```

Comment:
I would keep cdn to be as restrictive as possible not sure how much I trust jsdelivr.net